torch/lib/THD/master_worker/common/RPCType.hpp

```diff
@@ -77,79 +77,79 @@ struct type_traits {};
 
 template<>
 struct type_traits<char> {
-  static constexpr RPCType type = RPCType::CHAR;
+  static RPCType type;
```

Comment:
It's useful to leave comments in such situations to make sure that someone won't accidentally revert this change during a refactor (because `constexpr` really makes more sense)